.release/security-scan.hcl

```
@@ -2,20 +2,25 @@
 # SPDX-License-Identifier: BUSL-1.1
 
 container {
-  secrets {
-    all = false
-  }
+	local_daemon = true
+
+	secrets {
+		all = true
+    skip_path_strings = ["/website/content/"]
+	}
 
-  dependencies    = false
-  alpine_security = false
+  dependencies    = true
+  alpine_security = true
 }
 
 binary {
   go_modules = true
-  osv        = false
+  osv        = false # TODO: set to true when osv is resolved
```

So that we know who to poke about this:

```suggestion
  osv        = false # TODO(deniz): set to true when osv is resolved
```

---

This should be resolved, switched it to `true` in here https://github.com/hashicorp/nomad/pull/20465/commits/dfff95becaf6f28364e2203cd83ae0bb8574f997